discovery/community.py

```diff
@@ -218,6 +218,11 @@ def on_results(success):
 
         super(DiscoveryCommunity, self).initialize()
 
+    def unload_community(self):
+        super(DiscoveryCommunity, self).unload_community()
+        if self.bootstrap:
+            self.bootstrap.stop()
```

Comment:
I am now extending Bootstrap to use the Taskmanger. I think that there is no use for a start and that the stop is more of a cleanup. Is it OK to rename stop to cleanup and let the resolve_until_success schedule stuff using the TaskManager?